flang/lib/Lower/Allocatable.cpp

```diff
@@ -65,10 +127,10 @@ static void genAllocatableSetBounds(Fortran::lower::FirOpBuilder &builder,
 
 /// Generate runtime call to set the lengths of a character allocatable
 /// descriptor.
-static void genAllocatableInitCharRtCall(Fortran::lower::FirOpBuilder &builder,
-                                         mlir::Location loc,
-                                         const fir::MutableBoxValue &box,
-                                         mlir::Value len) {
+static void genAllocatableInitCharacter(Fortran::lower::FirOpBuilder &builder,
```

Comment:
I would rather keep the ugly `RtCall` in the functions doing runtime call just so that it's very clear we do not need to call this if we are not using it. In general, I should have added it to all the functions making runtime call here to make the distinctions very clear, even if a bit ugly. Now, if you really hate it, maybe having all the runtime calling function in a `runtime` namespace here could be a nicer solution to achieve the highlighting I wanted.

Reply:
This is one of four current genXyz functions that call Xyz functions from `flang/runtime/allocatable.h`.  There are ten more such functions that will eventually also need to be called.  It may be better to wait until more of these genXyz functions are implemented to decide what to do here.